source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,462 @@
+.. _minio-site-replication-overview:
+
+=========================
+Site replication overview
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Site replication configures multiple independent MinIO sites or clusters as replicas.
+When configured this way, refer to the replica sites as either "peer sites" or "sites".
```

Comment:
I think we can integrate this into the previous sentence and possibly drop one use of 'sites'